requirements.tests.txt

```diff
@@ -3,6 +3,7 @@ pytest==4.4.1
 freezegun==0.3.11
 pytest-cov==2.6.1
 pytest-postgresql==1.4.0
+mirakuru==1.1
```

Comment:
This seems to fix a crash when attempting to run tests.

`TypeError: descriptor '__subclasses__' of 'type' object needs an argument`

Reply:
Yeah, it was an unpinned dependency of pytest-postgresql, and they just release 2.0, which breaks in py2, among other issues.